### ISSUE TITLE
Problem with fontenc?

### ISSUE THREAD

**Issue by fabrivelas** [moderncv/moderncv#36]
When recompiling a cv from the beginning of the year and after updating moderncv to version 2021/01/21 v2.1.0 I got the following error message: 

> LaTeX Error: Option clash for package inputenc.

 Because I have a the following line in my code:
`\usepackage[latin1]{inputenc}`

After I commented the line 115 in **moderncv.cls**
`    \RequirePackage[utf8]{inputenc}`
 everything worked as before without a mistake.

**Comment by stephanlachnit** [comment]
Why don't you simply use utf8? There is literally no reason not to.

**Comment by fabrivelas** [comment]
When I tried converting the input tex file from latin1 to uft8, all the umlauts (ü,ä,ö) were unreadable.

**Comment by stephanlachnit** [comment]
Because your file is formatted wrongly (not utf8). UTF-8 has all the characters and is the way to go nowadays.

On Unix use iconv (https://manpages.debian.org/unstable/manpages/iconv.1.en.html), on Windows something like this (https://superuser.com/a/1163765).

**Comment by fabrivelas** [comment]
Yes, this happened with iconv. Or maybe my command (`iconv -f latin1 -t utf8 source.tex > target.tex`) was wrong. I have to check.... just checked. I used a wrong conversion, should have used `iconv -f ISO-8859-1 -t utf-8 source.tex > target.tex`. I checked the encoding with` file -I source.tex` and found that it was  ISO-8859-1 and not latin1. Thanks for helping so quickly. And thanks for the excellent package!
